README.md

```diff
@@ -1,4 +1,46 @@
 # Checkpoints
-[![stable](https://img.shields.io/badge/docs-latest-blue.svg)](https://invenia.github.io/Checkpoints.jl/stable)
+[![stable](https://img.shields.io/badge/docs-stable-blue.svg)](https://invenia.github.io/Checkpoints.jl/stable)
 [![build status](https://github.com/invenia/Checkpoints.jl/workflows/CI/badge.svg)](https://github.com/invenia/Checkpoints.jl/actions)
 [![coverage](https://codecov.io/gh/invenia/Checkpoints.jl/branch/main/graph/badge.svg)](https://codecov.io/gh/invenia/Checkpoints.jl)
+
+Checkpoints.jl allows packages to `register` checkpoints which can serialize objects to disk
+during the execution of an application program, if the application program `config`ures them.
+
+A minimal working example consists of the package:
+
+```julia
+module MyPackage
+
+using Checkpoints
+
+MODULE = "MyPackage"
+
+__init__() = Checkpoints.register(MODULE, ["foo", ])
+
+function foo(x)
+    checkpoint(MODULE, "foo", :data => 2x)
+    return 2x
+end
+
+end
+```
+
+and the application program:
+
+```julia
+using Checkpoints
+
+Checkpoints.config("MyPackage.foo", "./path/to/checkpoints")
+
+for i in 1:2
+    with_checkpoint_tags(:iteration => i) do
```

Comment:
so, if I provide multiple pairs, e.g. ` with_checkpoint_tags(:iteration => i, :tag2 => i+1) do`, I will end up with multiple subfolders?

```
./path/to/checkpoints/iteration=1/tag2=2/MyPackage/foo.jlso
./path/to/checkpoints/iteration=2/tag2=3//MyPackage/foo.jlso
```

Reply:
That's right. This is described in the docs, but do you think it should be in the README as well? I could add it to the package part which would signal that it is possible to use `with_checkpoint_tags` also in the package, not just the application.

Reply:
I've added multiple tags inside the package. Is this more informative, or too cluttered?